### ISSUE TITLE
Make the 'cant see the map' control visible by default

### ISSUE THREAD

**Issue by wrightmartin** [mysociety/fixmystreet#2410]
The "Can’t see the map? Skip this step" control only appears when tabbed over. 

This should be visible by default as it's not only a potential problem for keyboard navigation users

**Comment by dracos** [comment]
Dupe of #1360.
